src/license_manager_agent_ops.py

```diff
@@ -41,6 +43,14 @@ def _derived_pypi_url(self):
         return (f"https://{pypi_username}:{pypi_password}@"
                 f"{url}/simple/{self._PACKAGE_NAME}")
 
+    def setup_cache_dir(self):
+        cache_dir = Path("/var/cache/license-manager")
```

Comment:
We won't be using an env var to set this?

Reply:
Why not make it a configurable?

Reply:
James and I discussed this yesterday. We decided making it hard-coded in the charm removes the overhead of specifying some path that has to be validated and have permissions adjusted and such. We can instead, with the charm, just have a fixed place where the cache lives.

Do you see an advantage to having it configurable?

Reply:
Making it configurable has the advantage that we can change the location on disk easily. But we don't need that for now. @julianaklulo I'm fine with the code this way, do you see any problems that we might have here?

Reply:
I agree with you @heitorPB, we can keep it this way for now.